src/commands.ts

```diff
@@ -0,0 +1,34 @@
+import * as vscode from 'vscode'
+import { findMatchingTag } from './tagMatcher'
+import { parseTags } from './tagParser'
+
+export async function jumpToMatchingTag() {
+  const editor = vscode.window.activeTextEditor
+
+  if (!editor) {
+    return
+  }
+
+  const tagsList = parseTags(editor.document.getText())
+  const position = editor.selection.active
+  const positionOffset = editor.document.offsetAt(position)
+
+  const match = findMatchingTag(tagsList, positionOffset)
+
+  if (match && match.opening !== match.closing) {
```

Comment:
I think `if (match)` only is more accurate. Self closing tag is it's own opening tag, so I think it's fine to jump to the beginning, when this command is used. More importantly, the message now is a little misleading, saying `No matching tag was found` when the tag is self closing - it is it's own matching tag.

Reply:
Will do